micromobility-state-machine-diagram.svg

```diff
@@ -0,0 +1 @@
zm6.0218506 2.734375l1.4375 -6.90625l1.1875 0l-0.25 1.140625q0.65625 -0.75 1.171875 -1.015625q0.53125 -0.28125 1.125 -0.28125q0.65625 0 1.078125 0.34375q0.4375 0.328125 0.578125 0.953125q0.515625 -0.65625 1.09375 -0.96875q0.59375 -0.328125 1.234375 -0.328125q0.875 0 1.296875 0.40625q0.4375 0.40625 0.4375 1.15625q0 0.3125 -0.15625 1.046875l-0.921875 4.453125l-1.171875 0l0.9375 -4.5625q0.125 -0.5625 0.125 -0.8125q0 -0.328125 -0.21875 -0.515625q-0.203125 -0.203125 -0.59375 -0.203125q-0.53125 0 -1.078125 0.328125q-0.546875 0.3125 -0.859375 0.828125q-0.296875 0.515625 -0.515625 1.59375l-0.703125 3.34375l-1.1875 0l0.984375 -4.671875q0.109375 -0.46875 0.109375 -0.671875q0 -0.34375 -0.21875 -0.546875q-0.21875 -0.203125 -0.546875 -0.203125q-0.5 0 -1.0625 0.328125q-0.546875 0.3125 -0.890625 0.875q-0.34375 0.5625 -0.5625 1.625l-0.6875 3.265625l-1.171875 0zm11.102417 0l1.4375 -6.90625l1.1875 0l-0.25 1.140625q0.65625 -0.75 1.171875 -1.015625q0.53125 -0.28125 1.125 -0.28125q0.65625 0 1.078125 0.34375q0.4375 0.328125 0.578125 0.953125q0.515625 -0.65625 1.09375 -0.96875q0.59375 -0.328125 1.234375 -0.328125q0.875 0 1.296875 0.40625q0.4375 0.40625 0.4375 1.15625q0 0.3125 -0.15625 1.046875l-0.921875 4.453125l-1.171875 0l0.9375 -4.5625q0.125 -0.5625 0.125 -0.8125q0 -0.328125 -0.21875 -0.515625q-0.203125 -0.203125 -0.59375 -0.203125q-0.53125 0 -1.078125 0.328125q-0.546875 0.3125 -0.859375 0.828125q-0.296875 0.515625 -0.515625 1.59375l-0.703125 3.34375l-1.1875 0l0.984375 -4.671875q0.109375 -0.46875 0.109375 -0.671875q0 -0.34375 -0.21875 -0.546875q-0.21875 -0.203125 -0.546875 -0.203125q-0.5 0 -1.0625 0.328125q-0.546875 0.3125 -0.890625 0.875q-0.34375 0.5625 -0.5625 1.625l-0.6875 3.265625l-1.171875 0zm11.211792 -2.359375l1.1875 -0.078125q0 0.515625 0.15625 0.875q0.15625 0.359375 0.578125 0.59375q0.421875 0.21875 0.96875 0.21875q0.78125 0 1.171875 -0.3125q0.390625 -0.3125 0.390625 -0.734375q0 -0.3125 -0.234375 -0.578125q-0.234375 -0.28125 -1.171875 -0.671875q-0.9375 -0.40625 -1.1875 -0.578125q-0.4375 -0.265625 -0.671875 -0.625q-0.21875 -0.359375 -0.21875 -0.828125q0 -0.8125 0.65625 -1.390625q0.65625 -0.59375 1.828125 -0.59375q1.296875 0 1.96875 0.609375q0.6875 0.59375 0.71875 1.578125l-1.15625 0.078125q-0.03125 -0.625 -0.453125 -0.984375q-0.40625 -0.375 -1.171875 -0.375q-0.609375 0 -0.953125 0.28125q-0.328125 0.28125 -0.328125 0.609375q0 0.3125 0.296875 0.5625q0.1875 0.171875 1.0 0.53125q1.359375 0.578125 1.703125 0.921875q0.5625 0.53125 0.5625 1.3125q0 0.515625 -0.3125 1.015625q-0.3125 0.484375 -0.96875 0.78125q-0.640625 0.296875 -1.515625 0.296875q-1.203125 0 -2.046875 -0.59375q-0.84375 -0.59375 -0.796875 -1.921875zm5.2734375 5.015625l0 -0.859375l7.765625 0l0 0.859375l-7.765625 0zm8.599976 -2.65625l2.0 -9.546875l1.171875 0l-1.984375 9.546875l-1.1875 0zm3.2736206 -2.625q0 -2.015625 1.1875 -3.34375q0.984375 -1.09375 2.578125 -1.09375q1.25 0 2.015625 0.78125q0.765625 0.78125 0.765625 2.109375q0 1.1875 -0.484375 2.21875q-0.484375 1.015625 -1.375 1.5625q-0.890625 0.546875 -
```

Comment:
I think we need to move this file into the /modes folder

Reply:
Fixed in 9451cbf